concepts/05 Widgets/DataGrid/25 Sorting/20 API/1 Initial and Runtime Sorting.md

```diff
@@ -18,60 +19,215 @@ Rows are sorted according to the data source by default. Set the [sortOrder](/ap
     
     <!--HTML-->
     <dx-data-grid ... >
-        <dxi-column dataField="City"    [sortIndex]="1" sortOrder="asc"></dxi-column>
-        <dxi-column dataField="Country" [sortIndex]="0" sortOrder="asc"></dxi-column>
+        <dxi-column
+            dataField="City"
+            [sortIndex]="1"
+            sortOrder="asc">
+        </dxi-column>
+        <dxi-column
+            dataField="Country"
+            [sortIndex]="0"
+            sortOrder="asc">
+        </dxi-column>
     </dx-data-grid>
 
     <!--TypeScript-->
     import { DxDataGridModule } from "devextreme-angular";
-    // ...
+
     export class AppComponent {
         // ...
     }
+
     @NgModule({
         imports: [
             // ...
             DxDataGridModule
         ],
         // ...
     })
-    
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxDataGrid>
+            <DxColumn
+                :sort-index="1"
+                data-field="City"
+                sort-order="asc"
+            />
+            <DxColumn
+                :sort-index="0"
+                data-field="Country"
+                sort-order="asc"
+            />
+        </DxDataGrid>
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { DxDataGrid, DxColumn } from 'devextreme-vue/data-grid';
+
+    export default {
+        components: {
+            DxDataGrid,
+            DxColumn
+        }
+    }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { DataGrid, Column } from 'devextreme-react/data-grid';
+
+    class App extends React.Component {
+        render() {
+            return (
+                <DataGrid>
+                    <Column
+                        sortIndex={1}
```

Comment:
It may be a good idea to use `defaultSortIndex` in both cases since the sort index may be changed at runtime, and you do not handle this change in your code.